readme.md

```diff
@@ -270,7 +270,13 @@ Whether the process was killed.
 
 Type: `string | undefined`
 
-The signal that was used to terminate the process.
+The name of the signal that was used to terminate the process.
+
+#### signalDescription
+
+Type: `string | undefined`
```

Comment:
Should be start using
```suggestion
Type: `string?`
```

?

Reply:
We have other properties in the `README` that are optional but have more one than type:

```md
#### all

Type: `string | Buffer | undefined`
```

```md
#### stdin (same for stdout, stderr)

Type: `string | number | Stream | undefined`
```

Should those stay the same or be `string? | Buffer?`?

Reply:
Good point. Thinking more about this, it might be too convoluted and TypeScript doesn't even support `?` for return types. Lets forget it for now.